composer.json

```diff
@@ -9,16 +10,19 @@
     ],
     "minimum-stability": "stable",
     "require": {
-    	"php": ">=5.3.0",
-        "guzzlehttp/guzzle": "3.8.1"
+    	"php": ">=5.5.0",
```

Comment:
The adapter lib requires 5.4.8 so we need at least that and we should probably add that version to travis if we do.  The code should work though.